homeassistant/helpers/device_registry.py

```diff
@@ -180,6 +181,10 @@ async def _async_migrate_func(
                 # Introduced in 2022.2
                 for device in old_data["devices"]:
                     device["hw_version"] = device.get("hw_version")
+            if old_minor_version < 4:
+                # Introduced in 2022.2
```

Comment:
```suggestion
                # Introduced in 2023.1
```
Maybe `2023.2` as 2022 did not get `.1` version

Reply:
The comment should not have been there at all, removed now.